Solutions/CohesitySecurity/Playbooks/Cohesity_Restore_From_Last_Snapshot/azuredeploy.json

```diff
@@ -0,0 +1,375 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "title": "Restore From Last Snapshot",
+        "description": "This playbook restores the latest good Data Hawk (Helios) snapshot.",
+        "prerequisites": "Deploy the Cohesity configuration and function apps",
```

Comment:
@yinghuang123 : Can you please mention the complete prerequisites here as well, or else mention the link to the readme.md file for this playbook.

Rest everything fine with playbook

Reply:
Done. I added the link.

Reply:
Please check again because I see it. Maybe there's a glitch?